Makefile

```diff
@@ -0,0 +1,44 @@
+TEST_OPTIONS=-m unittest test
+DEBUGGER_PATH=/usr/lib64/python2.7/pdb.py
```

Comment:
On my box, the debugger is located in `/usr/lib/python2.7/pdb.py` (no 64).
I think we need another method to find it.